gems/manageiq-providers-amazon/manageiq-providers-amazon.gemspec

```diff
@@ -0,0 +1,17 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+require "manageiq/providers/amazon/version"
+
+Gem::Specification.new do |s|
+  s.name        = "manageiq-providers-amazon"
+  s.version     = ManageIQ::Providers::Amazon::VERSION
+  s.authors     = ["ManageIQ Developers"]
+  s.homepage    = "https://github.com/ManageIQ/manageiq"
+  s.summary     = "Amazon AWS Provider for ManageIQ"
+  s.description = "Amazon AWS Provider for ManageIQ"
+  s.licenses    = ["GPL-2.0", "Apache-2.0"]
+
+  s.files = Dir["{app,lib}/**/*"]
+
+  s.add_dependency("aws-sdk", ["~>2.2.19"])
```

Comment:
Nope, that's fine -- only things listed directly in the Gemfile are auto-required by default... Bundler assumes that gems take care of requiring their own dependencies as needed.